move/sources/canvas/token.move

```diff
@@ -382,10 +383,13 @@ module addr::canvas_token {
             let now = now_seconds();
             if (smart_table::contains(&canvas_.last_contribution_s, caller_addr)) {
```

Comment:
can we move this from being a smart table to being a regular table? IDK if @banool did benchmarking or not, but for O(1) lookups I'd assume table is faster than smart table (which also includes iteration)

Reply:
can i do the refactor in later pr? want to merge the feature pr first